content/components/icon.mdx

```diff
@@ -0,0 +1,216 @@
+|                                                       <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12" width="12" height="12"><path d="M6 0a6 6 0 110 12A6 6 0 016 0zm3 5H3v2h6z"></path></svg>                                                        | [`no-entry-fill`](https://primer.style/octicons/no-entry-fill-12)         | Indicate an ending or that the user is blocked and cannot continue                                                             |
+|     <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12" width="12" height="12"><path d="M1.757 10.243a6.001 6.001 0 118.488-8.486 6.001 6.001 0 01-8.488 8.486zM6 4.763l-2-2L2.763 4l2 2-2 2L4 9.237l2-2 2 2L9.237 8l-2-2 2-2L8 2.763z"></path></svg>     | [`x-circle-fill`](https://primer.style/octicons/x-circle-fill-12)         | For negative messaging to inform the user that an error has occurred as a result of an action or an action is unavailable      |
+|       <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12" width="12" height="12"><path d="M6 8.8c-.2 0-.4-.1-.5-.2L2.2 5.3c-.3-.3-.3-.8 0-1.1.3-.3.8-.3 1.1 0L6 6.9l2.7-2.7c.3-.3.8-.3 1.1 0 .3.3.3.8 0 1.1L6.6 8.5c-.2.2-.4.3-.6.3z"></path></svg>       | [`chevron-down`](https://primer.style/octicons/chevron-down-12)           | Indicates that a collapsible section is currently open                                                                         |
+|      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12" width="12" height="12"><path d="M4.7 10c-.2 0-.4-.1-.5-.2-.3-.3-.3-.8 0-1.1L6.9 6 4.2 3.3c-.3-.3-.3-.8 0-1.1.3-.3.8-.3 1.1 0l3.3 3.2c.3.3.3.8 0 1.1L5.3 9.7c-.2.2-.4.3-.6.3z"></path></svg>      | [`chevron-right`](https://primer.style/octicons/chevron-right-12)         | Indicates that a collapsible section is currently closed                                                                       |
+
+Maintaining the original size of icons is important for visual consistency, legibility, accessibility, and simplifying the design process. Resizing icons should be avoided.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/980622/209154753-14d8d1c9-0e9c-46e4-a392-43021a65a612.png"
+      role="presentation"
+      width="456"
+      alt="A pencil icon shown in their original 16px and 24px size."
+    />
+    <Caption>Keep icons in their original size.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/980622/209154759-7b1ef599-e4bd-4a75-869f-71de72329ddc.png"
+      role="presentation"
+      width="456"
+      alt="A pencil icon scaled up from 16px to 24px demonstrating how the stroke increased while upscaling. Another pencil icon is scaled down from 24px to 16px and demonstrates how the stroke became too thin."
+    />
+    <Caption>Don't resize icons.</Caption>
+  </Dont>
+</DoDontContainer>
+
+## System vs user-selectable
+
+There are two types of icons: `system icons` and `user-selectable` icons.
```

Comment:
Maybe the `icons` part should be in the backticks for `user-selectable` as well. So `user-selectable icons`?
This would than also apply to my below comments.

Reply:
Good spot I've changed it to `system` to be consistent with the title.